assets/script.js

```diff
@@ -225,5 +233,4 @@ function passFunction() {
   generateRandomRestaurant();
   tags.classList.add("has-text-warning");
   tags.classList.add("subtitle");
-  tags.classList.add("has-text-centered");
```

Comment:
This should not be removed from the script. The passed restaurants will not be centered